packages/storybook/src/theme.ts

```diff
@@ -1,5 +1,79 @@
+const breakpoints = ['640px', '768px', '1024px', '1280px']
+
+const baseColors = {
+  transparent: 'transparent',
+  black: '#000',
+  white: '#fff',
+  blue: [null, '#0067b6', '#0076d1', '#bfddf3', '#ebf6ff'],
+  green: [null, '#379a58', '#63db8b', '#dbffe7'],
+  yellow: [null, '#db9600', '#ffbe33', '#fff9eb'],
+  red: [null, '#c42020', '#e12f2f', '#ff8c8c', '#ffe5e5'],
+  sky: [null, '#f7f9fb', '#e3e9f0', '#cbd5e1', '#b3c2d3'],
+  ink: [null, '#1c384e', '#485f6f', '#65778b', '#8694a7'],
```

Comment:
Since ink and sky are a scale of greys (grays?) I was also thinking that these can be merged into a single grey scale, again like tailwind as their api is nice for consistency and have the convention that we go from lightest to darkest.  

This is unfortunately a difference between our current system but this convention might simplify things going forward.

Reply:
we can still have sky and ink as scales but move forward as references to the grey (or vice versa)

Reply:
Yea, that is worth considering. The only reason why I like breaking out sky and ink is because you have a separate scale for text and decorational elements. No text colors are going to come from sky and no borders or bgs are going to come from ink. Otherwise, you have to know where that break is in the scale of 8 different greys.

Reply:
I can be convinced otherwise though :)

Reply:
it's just that sky invokes the sense of blue and it may be unexpected when it comes out as a blue hued grey.  Maybe it's a differentiation between internal theming and the default example theme but it'd be nice to keep them similar.

Agree that knowing where to start is harder but probably no more arbitrary than ink3 -- we could know the convention is grey7-9 for text and grey1-3 for borders (normally).  Then if we want to swap out a warning for a secondary we know that a good start is `yellow.2` if we had a border of `grey.2` if our color scales are proportional in lightness and similarly the higher numbers are better for text in all colors.

Reply:
> Maybe it's a differentiation between internal theming and the default example theme but it'd be nice to keep them similar.

Yes, very good point on this. I'm good with going to one scale for grays.